client/src/Club.css

```diff
@@ -0,0 +1,28 @@
+
+  .custom-header {
+    background-color: #333;
+    color: white;
+    padding: 10px; /* Add padding to the entire header */
```

Comment:
Not a blocker but we don't really need comments on CSS properties (unless it's something really weird/confusing), you don't have to remove them here, just a head's up for the future to save some time